tests/QMLEngine/qml/BasicCompletedOfDynamicObjects.qml

```diff
@@ -0,0 +1,10 @@
+import QtQuick 2.0
+
+Rectangle {
+  color: "red"
+  id: item
+  
+  Component.onCompleted: {
+    Qt.createQmlObject( "import QtQuick 2.2\nItem { Component.onCompleted: parent.color = 'cyan'; }", item,"inlinecode1",__executionContext );
```

Comment:
It's a bummer, but we already have that issue with the createComponent method. We don't have yet something that would allow us to know which executionContext to use within createComponent and createQmlObject.

I've been trying to merge the realinheritance branch, and it seems to implement a variable that would give us the proper executionContext. But realinheritance is insanely hard to merge... some of the first commits are so large than github can't make a diff, and master has diverged so far from kde-master that if we did merge realinheritance as it is now, we'd break a lot of stuff for sure.